apps/dotcms-ui/src/app/api/services/dot-workflow-event-handler/dot-workflow-event-handler.service.ts

```diff
@@ -153,6 +153,7 @@ export class DotWorkflowEventHandlerService {
             delete data.environment;
             delete data.pushActionSelected;
         }
+        data['contentlet'] = {}; // needed for indexPolicy=WAIT_FOR
```

Comment:
Can we just add this to the interface?

Reply:
that is not under an interface because this data comes from the wizard that can collect any value.